content/en/docs/instrumentation/python/getting-started.md

```diff
@@ -265,21 +281,122 @@ service:
             processors: [batch]
 ```
 
-Then start the Docker container:
+Then run the docker command to acquire and run the collector based on this configuration:
 
-```sh
+```
 docker run -p 4317:4317 \
     -v /tmp/otel-collector-config.yaml:/etc/otel-collector-config.yaml \
     otel/opentelemetry-collector:latest \
     --config=/etc/otel-collector-config.yaml
 ```
 
-Install the OpenTelemetry Collector exporter:
+You will now have an OpenTelemetry Collector instance running locally.
+
+### Modify the code to export spans via OTLP
+
+The next step is to modify the code to send spans to the Collector via OTLP instead of the console.
 
-```sh
+To do this, install the OTLP exporter package:
+
+```
 pip install opentelemetry-exporter-otlp
 ```
 
-Finally, execute the following script:
+Then configure the exporter. By default, it will send to `locahost:4317`, which is what the collector listens on.
+
+```python
+# These are the necessary import declarations
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import (
+    BatchSpanProcessor,
+)
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import (
+    OTLPSpanExporter,
+)
+
+from random import randint
+from flask import Flask, request
+
+# Initialize tracing, but with OTLP
+provider = TracerProvider(resource=resource)
+processor = BatchSpanProcessor(OTLPSpanExporter())
+provider.add_span_processor(processor)
+trace.set_tracer_provider(provider)
```

Comment:
Same as above. distro sets up the trace pipeline already so this part is not doing anything.